fuzzing/instrum_opts.bzl

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fuzz test instrumentation options.
+
+Each fuzzing engine or sanitizer instrumentation recognized by the
+//fuzzing:cc_engine_instrumentation and //fuzzing:cc_engine_sanitizer
+configuration flag should be defined here.
+"""
+
+def _check_is_string_list(name, value):
```

Comment:
> just a suggestion since the current version is fine too: I think "list_of_strings" reads more naturally than "string_list".

Agreed this can be spelled differently, but I tend to favor "string list" since it is consistent with Starlark naming. For example, [`attr.string_list`](https://docs.bazel.build/versions/master/skylark/lib/attr.html#string_list).

> Also, maybe use "die" or "fail" instead of "check"? E.g., "is_list_of_strings_or_fail". I think that makes it more obvious what the side effects are of this function

Going with your other suggestion to `fail()` in the top-level call.

Reply:
Thanks for pointing out that Starlark already using string_list, SGTM